model/org.eclipse.birt.report.model/src/org/eclipse/birt/report/model/api/ReportDesignHandleImpl.java

```diff
@@ -378,19 +378,18 @@ public void setRefreshRate(int rate) {
 	 */
 	@Override
 	public SlotHandle getStyles() {
-		return getSlot(IReportDesignModel.STYLE_SLOT);
+		return getSlot(IInternalReportDesignModel.STYLE_SLOT);
 	}
 
 	/**
-	 * Gets all css styles sheet
+	 * Gets all CSS styles sheet
 	 *
 	 * @return each item is <code>CssStyleSheetHandle</code>
 	 */
-
-	public List getAllCssStyleSheets() {
+	public List<CssStyleSheetHandle> getAllCssStyleSheets() {
 		ReportDesign design = (ReportDesign) getElement();
-		List allStyles = new ArrayList();
-		List csses = design.getCsses();
+		List<CssStyleSheetHandle> allStyles = new ArrayList<CssStyleSheetHandle>();
+		List<?> csses = design.getCsses();
```

Comment:
The method is `	public final List<CssStyleSheet> getCsses() {` so you should not need casting and can use a for-each loop.

Reply:
Change is done.